### PR TITLE
Nits and fixups

### PR DESCRIPTION
**Pull request by niftynei**
Things found while working on the accountant plugin.

I have no idea how the wallet error made it past previous checks. Whoops.

The `listchannels` addition of the funding_outnum is extremely useful for backfilling missed channel open events.

### CODE REVIEW

**Review by rustyrussell — APPROVED** [review]
Ack 087ca31888e1df6004c8fc45061f9b45c6cda061

**Review by vincenzopalazzo — APPROVED** [review]
ack https://github.com/ElementsProject/lightning/pull/5016/commits/087ca31888e1df6004c8fc45061f9b45c6cda061

The CI error looks strange, and maybe it needs another run!

**Comment by cdecker** [comment]
Seems the JSON-RPC schema needs to be updated:

```
Additional properties are not allowed ('funding_outnum' was unexpected)
```

File: doc/schemas/listpeers.schema.json [hunk 1]
```diff
@@ -235,6 +235,10 @@
                   "type": "txid",
                   "description": "ID of the funding transaction"
                 },
+	        "funding_outnum": {
+		  "type": "u32",
+                  "description": "The 0-based output number of the funding transaction which opens the channel"
+		},
```
Comment (vincenzopalazzo) [inline doc/schemas/listpeers.schema.json:241]: Mh from CI there is a formatting problem here, maybe a `make check-doc` will call `jq` to auto formatting the json file

**Review by vincenzopalazzo — APPROVED** [review]

File: doc/schemas/listpeers.schema.json (hunk 1 above)
Comment (niftynei) [reply doc/schemas/listpeers.schema.json]: Yep that was the issue for this test. Nice catch!

**Review by vincenzopalazzo — APPROVED** [review]
ACK https://github.com/ElementsProject/lightning/pull/5016/commits/86c995879f8bd57d68f5a46e0720be762e4a5155

**Pull request merged**